dist/tools/randhex/randhex.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+bits="${1:-64}"
+if [ $((bits % 2)) -ne 0 ]; then
+	echo "${0##*/}: Amount of bits must be even" 1>&2
+	exit 1
+fi
```

Comment:
This sanity check can also be removed. In fact the entire code can probably be shortened to `tr -dc 'A-F0-9' < /dev/urandom | head -c 16` and moved to `sys/ssp/Makefile`. Not sure what you prefer.